src/ontology/mondo-ingest.Makefile

```diff
@@ -563,12 +563,18 @@ $(EXTERNAL_CONTENT_DIR)/%.robot.owl: $(EXTERNAL_CONTENT_DIR)/%.robot.tsv
 $(EXTERNAL_CONTENT_DIR)/nord.robot.tsv: $(TMPDIR)/nord.tsv config/external-content-robot-headers.json
 	mkdir -p $(EXTERNAL_CONTENT_DIR)
 	python ../scripts/add-robot-template-header.py $^ > $@
-.PRECIOUS: $(EXTERNAL_CONTENT_DIR)/nord.robot.owl
+.PRECIOUS: $(EXTERNAL_CONTENT_DIR)/nord.robot.tsv
 
-.PHONY: external-content-nord
+$(EXTERNAL_CONTENT_DIR)/nando-mappings.robot.tsv: $(MAPPINGSDIR)/mondo-nando.sssom.tsv
+	mkdir -p $(EXTERNAL_CONTENT_DIR)
+	python ../scripts/sssom_to_robot_template.py --inpath $^ --outpath $@
```

Comment:
Updated the make goal for creating robot template

Reply:
```suggestion
	python ../scripts/sssom_to_robot_template.py --inpath $< --outpath $@
```

for future proofing